docs/examples.rst

```diff
@@ -263,11 +263,11 @@ CybOX type.
 
     <cyboxCommon:HashListType>
         <cyboxCommon:Hash>
-            <cyboxCommon:Type xsi:type="cyboxVocabs:HashNameVocab-1.0">MD5</cyboxCommon:Type>
+            <cyboxCommon:Type>MD5</cyboxCommon:Type>
```

Comment:
I wanted that so bad, but I couldn't figure out how to do it! The object creation from dict is always initiated by `VocabString` (not the default subclass), which inspects the `xsi:type` key and resolves the appropriate subclass if found.  There really isn't any way (that I could think of) to have `VocabString` know which `VocabField` is calling it, so I can't try to create the default since I don't know what the default is.  If it's possible then I'd love this behavior.